txpipe/blinding.py

```diff
@@ -0,0 +1,157 @@
+from .base_stage import PipelineStage
+from .data_types import SACCFile
+import numpy as np
+import warnings
+
+class TXBlinding(PipelineStage):
+    """
+    Blinding the data vectors.
+
+    """
+    name='TXBlinding'
+    inputs = [
+        ('twopoint_data_UNBLINDED_RABID', SACCFile),
```

Comment:
Actually - we do need a suffix here, because otherwise the unblind file will be used for later stages.
Argh.  Actually, I think I need to think about this.

Also in the master branch we've change `twopoint_data` -> `twopoint_data_real`

Reply:
Okay, think the answer here is to change the suffix on the original file to `_raw` and then also add a null blinding stage that just copies `twopoint_data_real_raw` -> `twopoint_data_real` (and overwrites the raw file with an empty one, as is done now).

Then `raw` doesn't indicate that it definitely will be blinded later, but is neutral on the question.

Reply:
Are these now the correct suffixes? Is there a way to copy the file using the TXPipe object or should it just be with shutil e.g.?